spec/features/idv/in_person_spec.rb

```diff
@@ -402,6 +402,18 @@
       expect_in_person_gpo_step_indicator_current_step(t('step_indicator.flows.idv.get_a_letter'))
       click_button t('forms.verify_profile.submit')
 
+      # personal key
+      expect_in_person_step_indicator_current_step(t('step_indicator.flows.idv.secure_account'))
+      expect(page).to have_content(t('titles.idv.personal_key'))
+      deadline = nil
```

Comment:
It doesn't look like we do anything with `deadline`. Did I miss a usage, or do we need another spec here, or what?

Reply:
yep. i'll remove it. it is not needed for this test